src/frontend/src/lib/components/ui/Input.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+	import { Input as GixInput, IconClose } from '@dfinity/gix-components';
+	import { nonNullish, notEmptyString } from '@dfinity/utils';
+	import { type ComponentProps } from 'svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	type GixInputProps = ComponentProps<GixInput>;
+	type InputProps = {
+		showResetButton?: boolean;
+		resetButtonAriaLabel?: string;
+	};
+
+	let {
+		showResetButton,
+		resetButtonAriaLabel,
+		value = $bindable(),
+		...props
+	}: InputProps & GixInputProps = $props();
+
+	const reset = () => {
+		value = undefined;
+	};
+</script>
+
+<GixInput {...props} bind:value>
+	<svelte:fragment slot="inner-end">
+		{#if (typeof value === 'string' ? notEmptyString(value) : nonNullish(value)) && showResetButton}
```

Comment:
does it work simplyfing?

```suggestion
		{#if nonNullish(value) && notEmptyString(value.toString()) && showResetButton}
```

Reply:
any update here?

Reply:
Changed condition